devnet/README.md

```diff
@@ -23,6 +23,39 @@ docker-compose down
 
 > **_NOTE:_**  The devnet is not stateful, so subsequent runs will start from the genesis block.
 
+## Running Cross Client Devnet
+
+A cross-client devnet is network of gossamer node(s) and substrate node(s).
+
+Steps to run a two node cross client devnet
+
+- Create `genesis-spec.json`
+```
+./polkadot build-spec --disable-default-bootnode --dev > genesis-spec.json
+```
+
+- Edit `genesis-spec.json` as per your needs. Add extra authorities if you want to. In order to add extra authorities, you would want to edit "validatorCount", "minimumValidatorCount", "invulnerables", "stakers", and "session"."keys"
```

Comment:
What do you mean by `"session"."keys` at the end?

Reply:
```
"session": {
    "keys" : something
}
```